airbyte-integrations/connectors/source-salesforce/source_salesforce/source.py

```diff
@@ -50,7 +51,7 @@ class SourceSalesforce(ConcurrentSourceAdapter):
 
     message_repository = InMemoryMessageRepository(Level(AirbyteLogFormatter.level_mapping[logger.level]))
 
-    def __init__(self, catalog: Optional[ConfiguredAirbyteCatalog], config: Optional[Mapping[str, Any]], **kwargs):
+    def __init__(self, catalog: Optional[ConfiguredAirbyteCatalog], config: Optional[Mapping[str, Any]], state: AirbyteStateMessage, **kwargs):
```

Comment:
Based on https://github.com/airbytehq/airbyte/pull/33522/files#diff-95988f9ff4d7e4e1a8849df7d7453b3215d82ba5eeba981e7d995ba346973330R24, the input of the methods is `BaseSource.read_state`. This method returns a `TState` and not a `AirbyteStateMessage`

Reply:
👍 Thanks, fixed.